dbt_subprojects/solana/models/_sector/dex/pumpswap/pumpswap_solana_pools.sql

```diff
@@ -0,0 +1,45 @@
+{{
+  config(
+    schema = 'pumpswap_solana',
+    alias = 'pools',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
```

Comment:
```suggestion
    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.created_at')],
```

Reply:
since you rename in model, we have to reference target column name (target = model here)